analysis/study_definition.py

```diff
@@ -1,33 +1,191 @@
-from cohortextractor import StudyDefinition, patients, codelist, codelist_from_csv
+from cohortextractor import StudyDefinition, patients, codelist, codelist_from_csv, filter_codes_by_category
 
-chronic_cardiac_disease_codes = codelist_from_csv(
-    "codelists/opensafely-chronic-cardiac-disease.csv", system="ctv3", column="CTV3ID"
-)
-chronic_liver_disease_codes = codelist_from_csv(
-    "codelists/opensafely-chronic-liver-disease.csv", system="ctv3", column="CTV3ID"
-)
-salbutamol_codes = codelist_from_csv(
-    "codelists/opensafely-asthma-inhaler-salbutamol-medication.csv",
-    system="snomed",
-    column="id",
-)
-systolic_blood_pressure_codes = codelist(["2469."], system="ctv3")
-diastolic_blood_pressure_codes = codelist(["246A."], system="ctv3")
+from codelists import *
+
```

Comment:
We should specify the included population in the python usings `patients.satisfying`. Something like

```
 population=patients.satisfying(
        """
    
        AND (age >=18 AND age <= 110)
        AND has_follow_up 
        AND (sex = "M" OR sex = "F")
        """,
```

Reply:
Ok, will add.